raster/src/main/scala/geotrellis/raster/stitch/Stitcher.scala

```diff
@@ -33,3 +33,37 @@ object Stitcher {
   }
 
 }
+
+
+trait StitcherR[T <: CellGrid, R <: CellGrid] extends Serializable {
```

Comment:
Perhaps an upcoming PR, if we want to think through and talk out that issue
a bit before committing to that path/design pattern.

On Sat, Jan 23, 2016 at 12:15 PM, Eugene Cheipesh <notifications@github.com>
wrote:

> In raster/src/main/scala/geotrellis/raster/stitch/Stitcher.scala
> <https://github.com/geotrellis/geotrellis/pull/1230#discussion_r50624947>:
>
> > @@ -33,3 +33,37 @@ object Stitcher {
> >    }
> >
> >  }
> > +
> > +
> > +trait StitcherR[T <: CellGrid, R <: CellGrid] extends Serializable {
>
> Do you think that can happen in upcoming PR or it should be taken care of
> here?
>
> —
> Reply to this email directly or view it on GitHub
> <https://github.com/geotrellis/geotrellis/pull/1230/files#r50624947>.
>